src/main/java/bio/terra/service/dataset/flight/ingest/IngestUtils.java

```diff
@@ -353,8 +353,13 @@ public static String getParquetFilePath(String targetTableName, String flightId)
     return "parquet/" + targetTableName + "/" + flightId + ".parquet";
   }
 
-  public static String getSnapshotParquetFilePath(UUID snapshotId, String targetTableName) {
-    return "parquet/" + snapshotId + "/" + targetTableName + ".parquet";
+  public static String getRetrieveSnapshotParquetFilePath(UUID snapshotId, String targetTableName) {
+    return "parquet/" + snapshotId + "/" + targetTableName + "/*.parquet/*";
+  }
+
+  public static String getSnapshotSliceParquetFilePath(
+      UUID snapshotId, String targetTableName, String snapshotSliceName) {
+    return "parquet/" + snapshotId + "/" + targetTableName + "/" + snapshotSliceName + ".parquet";
```

Comment:
This pattern would be a breaking change - Is that okay? Or should I figure out a wildcard pattern that would work for both the old and new file path.

Reply:
Hmm...so would existing snapshots no longer work?  I _think_ it's ok but we may need to migrate?  Actually, as long as the preview endpoint still works I think we should be ok?